app/controllers/students/registrations_controller.rb

```diff
@@ -0,0 +1,76 @@
+class Students::RegistrationsController < Devise::RegistrationsController
+  before_filter :configure_sign_up_params, only: [:create]
+  # before_filter :configure_account_update_params, only: [:update]
+  
+ # GET /resource/sign_up
+  def new
+     super
+   end
+# def list
+#   @students=Student.all
+# end
+  # POST /resource
+   def create
+
+    
+     super
+     end
+
+  # GET /resource/edit
+  #  def edit
+  #  super
+  # end
+
+  # # PUT /resource
+  #  def update
+  #    super
+  #  end
+
+  # DELETE /resource
+  # def destroy
+  #   super
+  # end
+
+  # GET /resource/cancel
+  # Forces the session data which is usually expired after sign
+  # in to be expired now. This is useful if the user wants to
+  # cancel oauth signing in/up in the middle of the process,
+  # removing all OAuth session data.
+  # def cancel
+  #   super
+  # end
+
+   protected
+
+  # If you have extra params to permit, append them to the sanitizer.
+  # def configure_sign_up_params
+  #   devise_parameter_sanitizer.for(:sign_up) << :attribute
+  # end
+
+  # If you have extra params to permit, append them to the sanitizer.
+
+    def configure_sign_up_params
+
+      devise_parameter_sanitizer.for(:sign_up) << :enrollment_no
```

Comment:
This can be a method for push all parameter  in devise_parameter_sanitizer
Ex.  
`[:enrollment_no, :email, :first_name, ...... ] .each do |parameter|
  devise_parameter_sanitizer.for(:sign_up) << parameter
end`